src/pages/accounts/SignInPage.tsx

```diff
@@ -99,6 +125,7 @@ function SignInPage() {
 				<SignInButton disabled={emailErr} onClick={onSubmit}>
```

Comment:
Not sure if this is a priority at the moment, but we should definitely look into submitting the form on enter, so someone could potentially log in without the need for a cursor 👀

Reply:
Its funny because I definitely tried to hit enter to sign in like 4 times. I will make that change now.